README.md

```diff
@@ -1,8 +1,8 @@
 # Vol-E App
 
-Volume Explorer (Vol-E) is a browser based volume viewer built with React and WebGL (Three.js). This package wraps the [vole-core](https://github.com/allen-cell-animated/volume-viewer) library.
+Volume Explorer (Vol-E) is a browser based volume viewer built with React and WebGL (Three.js). This package wraps the [vole-core](https://github.com/allen-cell-animated/vole-core) library.
 
-For the latest stable release, please visit https://allen-cell-animated.github.io/website-3d-cell-viewer-release/
+For the latest stable release, please visit [https://volumeviewer.allencell.org](https://volumeviewer.allencell.org)
```

Comment:
@toloudis do we still want to redirect people to the GH pages build?

Reply:
I'd say no, there's no real reason to publicly link to the gh-pages version.  and the url here should probably be vole.allencell.org